Package.swift

```diff
@@ -23,7 +23,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-package-manager", .upToNextMajor(from: "0.5.0")),
+        .package(url: "https://github.com/apple/swift-tools-support-core.git", .upToNextMajor(from: "0.1.0")),
```

Comment:
I believe if we bump the version to `0.1.5` we'll get the fix for this :)

https://github.com/apple/swift-tools-support-core/commit/f34f419c73d39c830a006ff4c113449801f1fb4f#diff-8a8089dc1f441b3f42878901d077d13cR11

Thanks @lakpa !